### DISCUSSION TITLE
Encountered an error while upgrading to V10 from V8

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6070] [Q&A]
https://github.com/getredash/redash/releases/tag/v10.0.0

Followed the upgrade instruction above.
After step 6 :Stop Redash services: docker-compose stop server scheduler scheduled_worker adhoc_worker (you might need to list additional services if you defined them in your docker-compose.yml previously), got the error below:
ERROR: In file '/opt/redash/docker-compose.yml', service 'image' must be a mapping not a string.

Does anyone know what the solution is?
Also watched the video redash provided, seems like they did the same as I did when pasting "redash/redash:10.0.0.b50363".

**Comment by gaecoli** [comment]
> https://github.com/getredash/redash/releases/tag/v10.0.0
> 
> Followed the upgrade instruction above. After step 6 :Stop Redash services: docker-compose stop server scheduler scheduled_worker adhoc_worker (you might need to list additional services if you defined them in your docker-compose.yml previously), got the error below: ERROR: In file '/opt/redash/docker-compose.yml', service 'image' must be a mapping not a string.
> 
> Does anyone know what the solution is? Also watched the video redash provided, seems like they did the same as I did when pasting "redash/redash:10.0.0.b50363".

Show me your /opt/redash/docker-compose.yml

**Reply by qq20230530** [reply]
Thank you.
Retried and succeeded in upgrading to v10.
I might mistype something on the first try.
